src/pages/ReportSettingsPage.js

```diff
@@ -151,7 +151,7 @@ class ReportSettingsPage extends Component {
 
     render() {
         const shouldShowRename = !ReportUtils.isPolicyExpenseChat(this.props.report);
-        const shouldDisableRename = ReportUtils.isDefaultRoom(this.props.report)
+        const isDefaultOrArchivedRoom = ReportUtils.isDefaultRoom(this.props.report)
```

Comment:
NAB but I personally think the previous variable name was more descriptive and explained why we cared these checks.

Reply:
that's fair, I think it was confusing for me when actually used, i.e. a boolean where `!shouldDisableRename` was like a weird double negative in my head